conf/sssd/sssd.conf

```diff
@@ -0,0 +1,29 @@
+[sssd]
+config_file_version = 2
+domains = LDAP
+services = nss, pam, sudo, ssh
+
+[nss]
+filter_groups = root
+filter_users = root
+
+[pam]
+
+[sudo]
+
+[ssh]
+
+[domain/LDAP]
+id_provider = ldap
+auth_provider = ldap
+chpass_provider = ldap
+sudo_provider = ldap
+
+ldap_uri = ldaps://localhost:636
```

Comment:
Hmpf i insist that it would really be great it we could make it work without TLS because it's useless overhead for local-loop networking and sooner or later it's gonna create stupid issues like "certificate is not trusted" because of whatever reason, in turn destroying SSH connectivity for admins etc...

Or at least add a comment as to why exactly TLS is needed